Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -666,9 +666,7 @@ def start_download(self, torrentfilename=None, infohash=None, tdef=None, dconfig
                 d.get_def().get_trackers_as_single_tuple()))
             if new_trackers:
                 self.tribler_session.update_trackers(tdef.get_infohash(), new_trackers)
-            return d
```

Comment:
Was this removal intentional?

Reply:
oops. It slipped from another PR. That's why we do reviews...